frontend/src/components/form/TextListField.vue

```diff
@@ -66,14 +66,8 @@ export interface TextListFieldProps {
 
 const props = defineProps<TextListFieldProps>();
 
-onMounted(() => {
-  if (!value.value) {
-    // force initialize to empty array
-    value.value = [];
-  }
-});
-
 function create() {
+  if (!value.value) value.value = [];
```

Comment:
`value.value` would benefit from improved naming

Reply:
`ref()` weirdness, I'll think of something.. I don't know if I want to start calling every var `x: Ref` `xRef: Ref`. Maybe just `fieldValue` in this case